drivers/dummy/rasterizer_dummy.h

```diff
@@ -39,584 +39,584 @@
 
 class RasterizerSceneDummy : public RasterizerScene {
 public:
-	/* SHADOW ATLAS API */
+    /* SHADOW ATLAS API */
```

Comment:
Something went wrong here. IDE misconfiguration, or wrong version of clang-format? (We need 5.0).

Reply:
Once fixed, please squash the fixups into the last commit, as we don't really need a specific commit for those.

Reply:
Visual studio code wants to reformat everything, while msvc ide has my clang-format settings.

Reply:
Squashed.